packages/amplication-server/src/core/assistant/functions/createBlueprint.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "createBlueprint",
+  "description": "Create a blueprint.",
+  "parameters": {
+    "type": "object",
+    "properties": {
+      "name": {
+        "type": "string",
+        "description": "the name of the blueprint"
```

The description across the function isn't clear on the case
```suggestion
        "description": "The name of the blueprint"
```